db/redis.go

```diff
@@ -190,6 +190,12 @@ func (r *RedisDriver) unmarshal(cveID string, cveResult, cpeResult *redis.String
 	}, nil
 }
 
+// GetIdByCpeURI returns cve IDs by psuedo-CPE
```

Comment:
__[Misspell]__ "psuedo" is a misspelling of "pseudo" ([link](https://sider.review/gh/repos/54628930/pulls/126#issue-3915067))

<sub>posted by Sider</sub>